imagepipeline/src/main/java/com/facebook/imagepipeline/cache/BufferedDiskCache.java

```diff
@@ -125,6 +111,18 @@ public Boolean call() throws Exception {
   }
 
   /**
+   * Performs disk cache check synchronously.
+   * @param key
+   * @return true if the key is found in disk cache else false
+   */
+  public boolean syncDiskCheck(final CacheKey key) {
+    if(containsSync(key)){
```

Comment:
You're still missing a space before the opening bracket. Otherwise this looks good to me.